include/swift/IDE/IDEBridging.h

```diff
@@ -163,8 +163,10 @@ class BridgedResolvedLocVector {
 extern "C" {
 #endif
 
-/// Entry point to run the NameMatcher written in swift-syntax.
-/// 
+/// Low-level entry point to run the NameMatcher written in swift-syntax.
+///
+/// - Warning: The only caller of this should be `swift::ide::runNameMatcher`.
```

Comment:
Would it be worth moving this into the cpp file in that case?

Reply:
Good idea 👍🏽